libdnf/transaction/db/rpm.hpp

```diff
@@ -0,0 +1,83 @@
+/*
+Copyright (C) 2017-2020 Red Hat, Inc.
+
+This file is part of libdnf: https://github.com/rpm-software-management/libdnf/
+
+Libdnf is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+Libdnf is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+
+#ifndef LIBDNF_TRANSACTION_DB_RPM_HPP
+#define LIBDNF_TRANSACTION_DB_RPM_HPP
+
+
+#include "libdnf/utils/sqlite3/sqlite3.hpp"
+
+#include <memory>
+#include <vector>
+
+
+namespace libdnf::transaction {
+
+
+class Package;
```

Comment:
I suggest that Package class is not rpm package, but swdb package. We have to ensure that both Package classes are exportable into multiple languages.

Reply:
I don't understand the suggestion. Each Package lives in another module (libdnf.rpm.Package vs libdnf.transaction.Package) if that's what you're looking for.

Reply:
Ok, then it should be fine when it is in another module.